### PR TITLE
update docs: how to choose the backend

### PR DESCRIPTION
**Pull request by enguerran**
Regarding the question `Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)` at the end of the installation.

Please could you correct my english, I don't know if my english is correct enough?

### CODE REVIEW

File: docs/get-started.rst [hunk 1]
```diff
@@ -182,6 +182,11 @@ Make sure you have the system requirements listed in the
     cd kinto/
     make serve
 
+At the end of the installation, you will be asked which backend you would like to use:
+
+    $ Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)
+
+Choose the default (just push "enter"), when you will need another one, you will know it.
```
Comment (Natim) [inline docs/get-started.rst]: If you don't know, just push "enter" to choose the default Memory backend, when you will need another one you will know it :)

**Comment by Natim** [comment]
Thank you for your PR. It looks good.

File: docs/get-started.rst [hunk 2]
```diff
@@ -182,6 +182,11 @@ Make sure you have the system requirements listed in the
     cd kinto/
     make serve
 
+At the end of the installation, you will be asked which backend you would like to use:
```
Comment (almet) [inline docs/get-started.rst]: We should say "During the installation", because this might not be at the end at some point :)

Comment (almet) [inline docs/get-started.rst]: With restructured text, the way to have a code block is by either using `::` or by using the `code-block` directive, like this:


```rst

.. code-block:: bash

   some code here
```

File: docs/get-started.rst [hunk 3]
```diff
@@ -182,6 +182,11 @@ Make sure you have the system requirements listed in the
...
+
+    $ Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)
+
+If you don't know, just push "enter" to choose the default Memory backend, when you will need another one you will know it :)
```
Comment (almet) [inline docs/get-started.rst]: nitpick: try to wrap the lines to 80 chars max.

Comment (almet) [inline docs/get-started.rst]: nitpick: try to wrap the lines to 80 chars max.

Comment (almet) [inline docs/get-started.rst]: If you don't know what to select, just push "enter" to choose the [...].

Also, consider replacing "when you will need another one you will know it :)" by "You can always change your backend selection later on" ?

**Comment by almet** [comment]
Hi, and thanks a lot for taking the time to send these changes to us, they're very welcome :)

I've added a few comments in the pull request, let me know when you're ready so we can merge!

File: docs/get-started.rst (hunk 3 above)
Comment (enguerran) [inline docs/get-started.rst]: I guess we need to help the user to "change it later".
I suppose we can change the backends directly in `config/kinto.ini` (with [the help of the doc](https://kinto.readthedocs.org/en/latest/configuration/settings.html#backends)).

But is there a CLI to to get this question back? `Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)` And so have a wizard to setup postgres or redit.

Comment (Natim) [inline docs/get-started.rst]: You can ask for a new file creation with:

    kinto --ini new_file.ini init

Or you can remove the previous ``config/kinto.ini`` and restart ``kinto init``

**Comment by leplatrem** [comment]
:+1:

**Comment by enguerran** [comment]
@almet I updated the PR with your comments. See if it's ok.

Maybe I can add a note about https://github.com/Kinto/kinto/pull/485#discussion_r54877130 to help people changing the configuration file.

File: docs/get-started.rst [hunk 4]
```diff
@@ -182,6 +182,15 @@ Make sure you have the system requirements listed in the
...
 
+During the installation, you will be asked which backend you would like to use:
+
+::
+
+    $ Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)
+
+If you don't know, just push "enter" to choose the default Memory backend.
+You can always change your backend selection 
+`later on <https://kinto.readthedocs.org/en/latest/configuration/settings.html#backends>``.
```
Comment (almet) [inline docs/get-started.rst]: I believe this link is incorrect. rest links needs to end with an underscore. Something like:

```rst
`later on <https://kinto.readthedocs.org/en/latest/configuration/settings.html#backends>`_
```

You also have an extra backtick there (at the end of the line).

Comment (enguerran) [inline docs/get-started.rst]: What a syntax ;)

fixed in the next commit.

**Comment by almet** [comment]
Great, thanks a bunch for this :+1: !

**Pull request merged**